atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/benchmarks/table/ConsecutiveNarrowTable.java

```diff
@@ -79,13 +80,14 @@ public int getNumRows() {
     public void setup(AtlasDbServicesConnector conn) {
         this.connector = conn;
         services = conn.connect();
-        Benchmarks.createTable(getKvs(), getTableRef(), Tables.ROW_COMPONENT, Tables.COLUMN_NAME);
-        setupData();
+        if (!services.getKeyValueService().getAllTableNames().contains(getTableRef())) {
+            Benchmarks.createTable(getKvs(), getTableRef(), Tables.ROW_COMPONENT, Tables.COLUMN_NAME);
+            setupData();
+        }
     }
 
     @TearDown(Level.Trial)
     public void cleanup() throws Exception {
-        this.services.getKeyValueService().dropTable(getTableRef());
```

Comment:
Okay. I've confirmed that currently there aren't any benchmarks that modify the contents of a `CleanNarrowTable` or `DirtyNarrowTable` directly. Could we add some documentation that benchmarks must not mutate the table (or, if they do, must reset the table to its original state)?

Not sure I like the complexity (we're basically maintaining state across multiple benchmarks), but if it does shave an hour off running the benchmarks it might be worthwhile.